src/ast/enrichment/annotate_params.c

```diff
@@ -0,0 +1,68 @@
+/*
+* Copyright 2018-2019 Redis Labs Ltd. and Contributors
+*
+* This file is available under the Redis Labs Source Available License Agreement
+*/
+
+#include "../ast.h"
+#include "../../util/arr.h"
+#include "../../query_ctx.h"
+//------------------------------------------------------------------------------
+//  Annotation context - query parameters
+//------------------------------------------------------------------------------
+
+static AnnotationCtx *_AST_NewParamsContext(void) {
+	AnnotationCtx *param_crx = cypher_ast_annotation_context();
+	return param_crx;
+}
+
+static void _collect_query_params_map(const cypher_astnode_t *ast_exp, rax *parameter_map) {
+	cypher_astnode_type_t type = cypher_astnode_type(ast_exp);
+	// In case of identifier.
```

Comment:
Fix comment - parameter, not identifier.

Reply:
fixed